service/src/main/java/uk/gov/hmcts/reform/fpl/service/ManageHearingsService.java

```diff
@@ -349,6 +349,12 @@ public void addOrUpdate(Element<HearingBooking> hearingBooking, CaseData caseDat
         } else {
             caseData.addHearingBooking(hearingBooking);
         }
+
+        caseData.setHearingDetails(caseData.getHearingDetails().stream()
+            .sorted((hearing1, hearing2) ->
+                hearing2.getValue().getStartDate().compareTo(hearing1.getValue().getStartDate()))
```

Comment:
not sure if start date is null? Worth to find someone to verify the live data or add a null checking on it?